blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/UserContainerImpl.java

```diff
@@ -22,20 +26,23 @@
 public class UserContainerImpl extends BlueUserContainer {
 
     private final Reachable parent;
+    private final BlueOrganization organization;
 
-    public UserContainerImpl(@Nonnull Reachable parent) {
+    public UserContainerImpl(@Nonnull BlueOrganization organization, @Nonnull Reachable parent) {
         this.parent = parent;
+        this.organization = organization;
     }
 
     public UserContainerImpl() {
         this.parent = null;
+        this.organization = OrganizationFactory.getInstance().list().iterator().next();
```

Comment:
... see comment about `getDefault()`, it would simplify/consolidate the different usages like this.